datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -9,31 +9,29 @@
 
 from datadog_checks.dev.tooling.constants import get_root
 
-from ...utils import complete_valid_checks, get_readme_file, get_valid_integrations, load_manifest, read_readme_file
+from ...testing import process_checks_option
+from ...utils import complete_valid_checks, get_readme_file, load_manifest, read_readme_file
 from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
 
 IMAGE_EXTENSIONS = {".png", ".jpg"}
 
 
 @click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate README.md files')
 @click.pass_context
-@click.argument('integration', autocompletion=complete_valid_checks, required=False)
-def readmes(ctx, integration):
+@click.argument('check', autocompletion=complete_valid_checks, required=False)
+def readmes(ctx, check):
     """Validates README files
 
-    If `check` is specified, only the check will be validated,
-    otherwise all README files in the repo will be.
+    If `check` is specified, only the check will be validated, if check value is 'changed' will only apply to changed
+    checks, 'all' or empty `check` value will validate all README files.
```

Comment:
This may be worth adding to the other commands as well since we can now have special values for `check`